internal/changelog/entry.go

```diff
@@ -42,12 +44,12 @@ func EntryFromFragment(f fragment.File) Entry {
 		},
 	}
 
-	if f.Fragment.Pr > 0 {
-		e.LinkedPR = []int{f.Fragment.Pr}
+	if len(f.Fragment.Pr) > 0 {
+		e.LinkedPR = []string{f.Fragment.Pr}
 	}
 
-	if f.Fragment.Issue > 0 {
-		e.LinkedIssue = []int{f.Fragment.Issue}
+	if len(f.Fragment.Issue) > 0 {
```

Comment:
```suggestion
	if f.Fragment.Issue != "" {
```

Reply:
Will do, any particular reason for not using `len`? The logic appears identical in this case.

Reply:
No change in behaviour but checking string length is not idiomatic in Go.